### PR TITLE
TST: add test for numpy ops, esp. nanmin/max bug for np<1.13

### LINKED ISSUE #8383: nanmin no longer works with series
Using NumPy's nanmin on a pandas Series worked as expected, now however the Series is returned with each element set to the minimum, rather than just getting the minimum. Not sure if this is a Numpy of Pandas issue.

```
np.nanmin(pds.Series([1,2,3,4]))
Out[14]: 
0    1
1    1
2    1
3    1
dtype: int64

np.min(pds.Series([1,2,3,4]))
Out[15]: 1
```

### PR DESCRIPTION
**Pull request by topper-123**
- [x] closes #8383
- [x] tests added / passed
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

The bug in #8383 concerns np.nanmin and np.nanmax only, don't know if I'm testing too much.

### CODE REVIEW

File: pandas/tests/test_nanops.py [hunk 1]
```diff
@@ -15,6 +16,7 @@
 import pandas.util._test_decorators as td
 
 use_bn = nanops._USE_BOTTLENECK
+np_version113 = LooseVersion(np.__version__) >= LooseVersion('1.13')
```
Comment (jreback) [inline pandas/tests/test_nanops.py]: import from pandas
we already define this

File: pandas/tests/test_nanops.py [hunk 2]
```diff
@@ -1015,3 +1017,25 @@ def test_use_bottleneck():
...
+    (np.nanmean, True, 2.5),
+    (np.median, True, 2.5),
+    (np.nanmedian, True, 2.5),
+    (np.min, True, 1),
+    (np.nanmin, False, 1),
+    (np.max, True, 4),
+    (np.nanmax, False, 4),
+])
+def test_numpy_ops(numpy_op, passes_all_numpy_version, expected):
+    # GH8383
```
Comment (jreback) [inline pandas/tests/test_nanops.py]: you can just make 2 tests
1 for numpy >= 1.13 and 1 for < 1.13 which assert those functions which exist there

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by topper-123** [comment]
Comments addressed. Not sure if I understood your second, but please look at changed versions.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
lgtm ping on green

**Comment by topper-123** [comment]
all green.

**Pull request merged**